bridge/src/main/java/com/bitwarden/bridge/model/AddTotpLoginItemDataJson.kt

```diff
@@ -10,7 +10,7 @@ import kotlinx.serialization.Serializable
  * @param totpUri A TOTP code URI to be added to the Bitwarden app.
  */
 @Serializable
-data class AddTotpLoginItemDataJson(
+internal data class AddTotpLoginItemDataJson(
```

Comment:
I realized when working on this PR that these are just an implementation detail of the encryption. 

I love being able to use `internal` 😎